packages/redaction/src/generic/__tests__/redact-string.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { redactString } from "../redact-string";
+
+describe("redactString", () => {
+  it("redacts emails correctly", () => {
+    expect(redactString("test@example.com")).toMatchInlineSnapshot(
+      `"____@_______.com"`
+    );
+  });
+
+  it("redacts URLs correctly", () => {
+    expect(redactString("https://example.com")).toMatchInlineSnapshot(
+      `"https://redacted.com/"`
+    );
+  });
+
+  it("redacts ISO 8601 timestamps correctly", () => {
+    expect(redactString("2024-01-01T00:00:00.000Z")).toMatchInlineSnapshot(
+      `"1970-01-01T00:00:00.000Z"`
```

Comment:
Should we redact to this to a more recent date like the start of the year or month? I'm imagining something like http://myapp.com/orders?year=2024&month=5 and then we redact everything to be in 1970 so the page shows as empty.